reflex-dom-core/src/Reflex/Dom/Builder/Static.hs

```diff
@@ -288,7 +288,7 @@ instance SupportsStaticDomBuilder t m => DomBuilder t (StaticDomBuilderT t m) wh
     StaticDomBuilderT $ do
       let shouldEscape = elementTag `Set.notMember` noEscapeElements
       (result, innerHtml) <- lift $ lift $ runStaticDomBuilderT child $ StaticDomBuilderEnv shouldEscape Nothing
-      attrs0 <- foldDyn applyMap (cfg ^. initialAttributes) (cfg ^. modifyAttributes)
+      attrs0 <- foldDyn applyMap (Map.insert "data-ssr" "" $ cfg ^. initialAttributes) (cfg ^. modifyAttributes)
```

Comment:
I think this insertion should be done in a separate monad transformer, not in Static.

Reply:
For my own understanding, why is this?

Reply:
Perhaps because `data-ssr` is hydration-specific but Static is not?

Reply:
BTW (FWIW): This line was introduced in the "Revert" commit so it's how it used to be at least.

Reply:
Right; this never should have been here, because it makes the static renderer not produce the HTML the user tells it to.